pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -133,19 +149,31 @@ func (c *DefaultIstioPerformer) Uninstall(kubeClientSet kubernetes.Client, log *
 	if err != nil {
 		return err
 	}
-	log.Info("Istio namespace deleted")
+	logger.Info("Istio namespace deleted")
 	return nil
 }
 
-func (c *DefaultIstioPerformer) Install(kubeConfig, manifest string, logger *zap.SugaredLogger) error {
-	istioOperator, err := extractIstioOperatorContextFrom(manifest)
+func (c *DefaultIstioPerformer) Install(kubeConfig, istioChart, requestedVersion string, logger *zap.SugaredLogger) error {
+	logger.Info("Starting Istio installation...")
+
+	version, err := istioctl.VersionFromString(requestedVersion)
+	if err != nil {
+		return errors.Wrap(err, "Error parsing version")
+	}
+
+	logger.Info("Requested istio version: " + version.String())
```

Comment:
May this log be the part of `resolver.GetCommander(version)` function?  Wouldn't it be reasonable to log it every in that place?

Reply:
It is done there, but with debug level, otherwise these is redundant, as it logs more or less the same. To be decided.

Reply:
Applied